api/python/provisioner/commands/setup_provisioner.py

```diff
@@ -1790,12 +1792,29 @@ def _run(self, nodes, **kwargs):  # noqa: C901 FIXME
         logger.info("Configuring provisioner logging")
         for state in [
             'components.system.prepare',
-            'components.provisioner.config'
         ]:
             ssh_client.cmd_run(
                 f"salt-call state.apply {state}",
                 targets=master_targets
             )
+        
+        # Seperation of variable to make flake8 happy
+        provisioner_pillar = {
+            'pillar': {
+                'provisioner': {
+                    'cluster': {
```

Comment:
generally I think we would need to re-think usage of inline pillar: bad that it sticks our logic much to api so states become more and more not self-contained.
So, that change looks pretty good for now. But will likely be re-thought later.